tests/Controller/AdminResettingControllerTest.php

```diff
@@ -13,20 +13,88 @@
 
 namespace Sonata\UserBundle\Tests\Controller;
 
+use FOS\UserBundle\Model\UserManager;
 use PHPUnit\Framework\TestCase;
+use Sonata\AdminBundle\Admin\Pool;
 use Sonata\UserBundle\Controller\AdminResettingController;
+use Symfony\Bundle\TwigBundle\TwigEngine;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
 
 class AdminResettingControllerTest extends TestCase
 {
     private $controller;
 
+    private $container;
+
+    private $twig;
+
+    private $userManager;
+
+    private $adminPool;
+
     protected function setUp(): void
     {
         $this->controller = new AdminResettingController();
+        $this->container = $this->getMockBuilder(ContainerBuilder::class)->getMock();
```

Comment:
please use `createMock()` here (and below), probably you use another (older) PHPUnit version.

This should fix the tests

Reply:
why? `createMock` is deprecated isn't it ?

Reply:
`getMock` is Deprecated IIRC

Reply:
ok :+1: